src/components/pages/Offer/Offer/OfferDetails/OfferForm/OfferForm.jsx

```diff
@@ -608,37 +627,46 @@ const OfferForm = ({
             <CheckboxInput
               SvgElement={VisualDisabilitySvg}
               checked={formValues.visualDisabilityCompliant}
+              isError={!!getErrorMessage('disabilityCompliant')}
```

Comment:
```suggestion
              isError={getErrorMessage('disabilityCompliant') !== undefined}
```

On s'était dit la dernière fois qu'on évitait la notation `!!` parce qu'elle n'était pas très lisible non ?

Reply:
ha bon ?
Pour moi c'est un bon moyen de casté en boolean.
Tu doit faire reference à la discution ou je disait que c'etait pas mal d'éviter les condition: `if (!isValid)` et de préférer `if (isNotValid)`.
Peut etre `Bool(getErrorMessage('disabilityCompliant'))` si tu à eu du mal à lire cette ligne.

Reply:
Ah pourquoi pas le `Bool()` en effet :)